src/v2/controllers/FormController.ts

```diff
@@ -209,7 +210,10 @@ export default Controller.extend({
     } else {
       error = new ConfigError(`Invalid action selected: ${actionPath}`);
       this.options.settings.callGlobalError(error);
-      await this.showFormErrors(this.formView.model, error, this.formView.form);
+      // only show form errors when not intentionally ignored from actionParams
+      if (!actionParams[ACTION_PARAMS_IGNORE_FORM_ERRORS]) {
+        await this.showFormErrors(this.formView.model, error, this.formView.form);
+      }
```

Comment:
How about we put a logger in a `else` here

Reply:
Looks like global error/log should be tracked [here](https://github.com/okta/okta-signin-widget/pull/3747/files#diff-72a0e868d27f4dfd26d36bcba1d860aa25999401fa00788f3ac694304043dd4dR212) already

Reply:
Can we change the logic here to `return` early if `ignoreFormErrors=true` instead of the negative logic?

Reply:
@shawyu-okta Updated